src/components/Register.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+function Register() {
+  const [newEmail, setNewEmail] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [error, setError] = useState(null);
+
+  async function registerForm(e) {
```

Comment:
would be good to use naming convention for event handlers here. function name for handlers should start with the word "handle" so "handleSubmit" or "handleRegistration" would work.